### ISSUE TITLE
Integer number_of_threads passed to string without any conversion

### ISSUE THREAD

**Issue by sebastientourbier** [connectomicslab/connectomemapper3#44]
As reported @ https://neurostars.org/t/connectome-mapper-3/16856, this raises the following error:
```
User: root
Group: root
> BIDS dataset: /bids_dir
> Subjects to analyze : ['0010']
> Set $FS_LICENSE which points to FreeSurfer license location (BIDS App)
  ... $FS_LICENSE : /bids_dir/code/license.txt
  * Number of subjects to be processed in parallel set to one (sequential run)
  * Number of parallel threads set to one (total of cores: 23)
  * OMP_NUM_THREADS set to 1 (total of cores: 23)
> Sessions to analyze : ['ses-1', 'ses-2']
> Process subject sub-0010 session ses-1
WARNING: rewriting config file /output_dir/cmp/sub-0010/ses-1/sub-0010_ses-1_anatomical_config.ini
... Anatomical config created : /output_dir/cmp/sub-0010/ses-1/sub-0010_ses-1_anatomical_config.ini
WARNING: rewriting config file /output_dir/cmp/sub-0010/ses-1/sub-0010_ses-1_diffusion_config.ini
... Diffusion config created : /output_dir/cmp/sub-0010/ses-1/sub-0010_ses-1_diffusion_config.ini
... Running pipelines : 
        - Anatomical MRI (segmentation and parcellation)
        - Diffusion MRI (structural connectivity matrices)
INFO: functional pipeline not performed
Traceback (most recent call last):
  File "/app/connectomemapper3/run.py", line 497, in <module>
    number_of_threads=number_of_threads)
  File "/app/connectomemapper3/run.py", line 73, in create_cmp_command
    return ' '.join(cmd)
TypeError: sequence item 14: expected str instance, int found
```

**Comment by sebastientourbier** [comment]
Fixed by commit da16620

**Comment by sebastientourbier** [comment]
I close this issue as the new docker image seems to work (https://neurostars.org/t/connectome-mapper-3/16856/5).
